Engine/DataFeeds/DataFeedPacket.cs

```diff
@@ -66,7 +66,10 @@ public SubscriptionDataConfig Configuration
         /// <param name="configuration">The subscription configuration that produced this data</param>
         /// <param name="isSubscriptionRemoved">Reference to whether or not the subscription has since been removed, defaults to false</param>
         public DataFeedPacket(ISecurityPrice security, SubscriptionDataConfig configuration, IReadOnlyRef<bool> isSubscriptionRemoved = null)
-            : this(security, configuration, new List<BaseData>(), isSubscriptionRemoved)
+            : this(security,
+                configuration,
+                new List<BaseData>(1), // performance: by default the list has 0 capacity, so lets initialize it with at least 1
```

Comment:
nice, although I wonder how it will grow when starting with 1. I believe the default behavior is to resize the list to 4 on the first addition, so it _may_ be better to start the list with a size of 4, BICBW.

Reply:
Your right default behavior is 4, lets trust default and set it to 4 👍